### PR TITLE
kraken: fetchMinOrderSizes: ensure parseJsonResponse is restored

### PR DESCRIPTION
**Pull request by mkutny**

### CODE REVIEW

**Comment by kroitor** [comment]
Oh, thanks for fixing it!

File: js/kraken.js [hunk 1]
```diff
@@ -209,9 +209,16 @@ module.exports = class kraken extends Exchange {
     }
 
     async fetchMinOrderSizes () {
-        this.parseJsonResponse = false;
-        let html = await this.zendeskGet205893708WhatIsTheMinimumOrderSize ();
-        this.parseJsonResponse = true;
+        let html;
```
Comment (kroitor) [inline js/kraken.js]: this one is missing an undefined rhs value )

Comment (kroitor) [reply js/kraken.js]: Let me know, if you want me to fix it.

Comment (mkutny) [reply js/kraken.js]: Nope. I'll fix the linter.

Comment (mkutny) [reply js/kraken.js]: Ok, fixed. Let me know if you want separate linter commits for such cases.

Comment (kroitor) [reply js/kraken.js]: I'm ok if they go in the same pr, as they are related anyways, and they are too small to detach them, I think.

**Pull request merged**